src/seagap/cli.py

```diff
@@ -0,0 +1,78 @@
+"""The main command line interface for seagap"""
+from typing import Any, Dict, Optional
+
+import typer
+
+from . import package_name
+from .configs.main import Configuration, load_configuration
+from .harmonic_mean import sv_harmonic_mean
+from .utilities.io import _get_filesystem
+
+app = typer.Typer(name=package_name)
+
+
+def load_files(config: Configuration) -> Dict[str, Any]:
+    all_files_dict = {}
+    for k, v in config.solver.input_files.dict().items():
+        typer.echo(f"Loading {k} at {v.get('path')}")
+        path = v.get("path", "")
+        storage_options = v.get("storage_options", {})
+
+        fs = _get_filesystem(path, storage_options)
+        if "**" in path:
+            all_files = fs.glob(path)
+        else:
+            all_files = path
+
+        all_files_dict.setdefault(k, all_files)
+    return all_files_dict
+
+
+def main(config: Configuration, all_files_dict: Dict[str, Any]):
+    import pandas as pd
+
+    # Read sound speed
+    svdf = pd.read_csv(
+        all_files_dict["sound_speed"],
+        delim_whitespace=True,
+        header=None,
+        names=["dd", "sv"],
```

Comment:
These column names also appeared in another PR. It would probably be good to define constant(s) for these names in a shared module. E.g.
```
COLUMN_DD = "dd"
COLUMN_SV = "sv"
COLUMNS = [COLUMN_DD, COLUMN_SV]
```